src/Command/EnableComposerAutoloadingCommand.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\Command;
+
+use Laminas\Cli\CommandListenerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Event\ConsoleEvent;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+use function file_get_contents;
+use function file_put_contents;
+use function json_decode;
+use function json_encode;
+
+use const JSON_PRETTY_PRINT;
+use const JSON_UNESCAPED_SLASHES;
+use const JSON_UNESCAPED_UNICODE;
+
+final class EnableComposerAutoloadingCommand extends Command implements CommandListenerInterface
+{
+    /** @var string */
+    protected static $defaultName = 'mvc:module:enable-autoloading';
+
+    protected function configure(): void
+    {
+        $this->setName(self::$defaultName);
+        $this->setDescription('Enables PSR-4 autoloading for module');
+        $this->addOption(
+            'module',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'Module name to enable'
+        );
+        $this->addOption(
+            'dir',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'Directory with modules'
+        );
+        $this->addOption(
+            'mode',
+            null,
+            InputOption::VALUE_OPTIONAL,
+            'Production or development mode'
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $module = $input->getOption('module');
+        $dir = $input->getOption('dir');
+
+        $mode = $input->getOption('mode');
+        if ($mode === null) {
```

Comment:
Since you're not validating arguments, perhaps this should be `if (! in_array($mode, ['Production', 'Development'], true)) {`?

Reply:
This is PoC, and there is missing validation in that command.
We are not also asking for other options if these are not provided.
I'd like to have all options optional, so if not provided there will be a question (interactive) - I've done it the `mvc:module:create` command, in the other just minimal stuff to get it working with chains.